webapp/src/serial.tsx

```diff
@@ -51,10 +52,12 @@ export class Editor extends srceditor.Editor {
         }
         this.isVisible = b
         if (this.isVisible) {
+            this.processQueuedMessages()
```

Comment:
I think you want to move this call in ``startRecording``

Reply:
The editor is always recording when it's opened, so this should be fine as-is.